docs/potts.rst

```diff
@@ -25,29 +31,42 @@ lattice and the simulation.
 
 
 
-This section appears at the beginning of the configuration file. Line
-``<Dimensions x="101" y="101" z="1"/>`` declares the dimensions of the
-lattice to be``101 x 101 x 1``, *i.e.*, the lattice is two-dimensional and
-extends in the ``xy`` plane. The basis of the lattice is ``0`` in each
-direction, so the ``101`` lattice sites in the ``x`` and ``y`` directions have
-indices ranging from ``0`` to ``100``. ``<Steps>1000</Steps>`` tells CompuCell how
-long the simulation lasts in MCS. After executing this number of steps,
-CompuCell can run simulation at zero temperature for an additional
-period. In our case it will run for ``<Anneal>10</Anneal>`` extra steps.
-``FluctuationAmplitude`` parameter determines intrinsic fluctuation or
-motility of cell membrane.
+This section appears at the beginning of the configuration file. 
+The line reading ``<Dimensions x="101" y="101" z="1"/>`` declares the dimensions of the
+lattice to be 101 by 101 by 1 pixels. 
+Since z is small, this lattice is two-dimensional. 
+Lattice sites are 0-indexed, which means that we count them from 0 to 100 when
+accessing them in Python. 
+
+.. _anneal:
+
+``<Steps>1000</Steps>`` tells CompuCell how
+long the simulation lasts in Monte Carlos Steps (MCS). After executing this number of steps, CompuCell can simulate with zero temperature for an additional
+period. In our case, it will run for ``<Anneal>10</Anneal>`` extra steps.
+
+.. _fluctuation-amplitude:
+
+Fluctuation Amplitude (Temperature)
+-------------------------------------------------------------------
+
+The ``FluctuationAmplitude``/``Temperature`` parameter determines the intrinsic fluctuation or
+motility of each cell membrane.
+Try setting a high ``Temperature`` value in XML. 
+You should see much more activity from pixel changes.
+Conversely, a low ``Temperature`` will 
 
 .. note::
 
    ``FluctuationAmplitude`` is a ``Temperature``
-   parameter in classical GGH model formulation. We have decided to use
-   ``FluctuationAmplitude`` term instead of temperature because using word
-   ``Temperature`` to describe intrinsic motility of cell membrane was quite
+   parameter in classical GGH Model formulation. We have decided to use
+   ``FluctuationAmplitude`` term instead of temperature because using the word
+   ``Temperature`` to describe the intrinsic motility of cell membranes was quite
    confusing.
 
 In the above example, fluctuation amplitude applies to all cells in the
 simulation. To define fluctuation amplitude separately for each cell
-type we use the following syntax:
+type, we use the following syntax. 
+Each value must be non-negative.
```

Comment:
Is the XML below this line for <FluctuationAmplitude> deprecated? I haven't seen it elsewhere.